src/client/styles/footer.scss

```diff
@@ -16,19 +16,16 @@
  *  limitations under the License
  *
  */
- 
+
 footer {
   max-width: 944px;
```

Comment:
I wonder if some 'magic numbers' like this should be variables?

Reply:
Sure! Let's do it in a separate PR though.